src/macros.rs

```diff
@@ -0,0 +1,72 @@
+//! Warning: extreme macros
+
+// TODO visibility
+// Usage:
+// lua_convert! {
+//
+// struct Point {
+//     x: i32,
+//     y: i32
+// };
+//
+// struct Bar {
+//     name: String,
+//     point: Point
+// }
+//
+// }
+macro_rules! lua_convertible {
+    (  $(#[$attr:meta])*
+       struct $name:ident { $($fname:ident : $ftype:ty, )+  }  ) => {
```

Comment:
The indentation here is a little confusing, at first I though there were multiple arms, but brace highlighting tells me there is just one. Perhaps you should indent after this line to better illustrate that?